forest/tests/config_tests.rs

```diff
@@ -121,3 +122,50 @@ fn test_config_env_var() {
 
     assert!(expected_config == actual_config);
 }
+
+#[test]
+fn test_download_location_of_proof_parameter_files_env() {
+    let tmp_dir = TempDir::new().unwrap();
+
+    Command::cargo_bin("forest")
+        .unwrap()
+        .env("FIL_PROOFS_PARAMETER_CACHE", tmp_dir.path())
+        .arg("fetch-params")
+        .arg("--keys")
+        .assert()
+        .success();
+
+    let list_files = read_dir(tmp_dir.path()).unwrap();
+
+    assert!(list_files.count() > 0);
```

Comment:
probably `!list_files.is_empty()` would be more idiomatic here and in the second test

Reply:
There is no such method for that type